clientv3/integration/kv_test.go

```diff
@@ -251,3 +253,31 @@ func TestKVDelete(t *testing.T) {
 		t.Fatalf("gresp.Kvs got %+v, want none", gresp.Kvs)
 	}
 }
+
+func TestKVCompact(t *testing.T) {
+	defer testutil.AfterTest(t)
+
+	clus := integration.NewClusterV3(t, &integration.ClusterConfig{Size: 3})
+	defer clus.Terminate(t)
+
+	kv := clientv3.NewKV(clus.RandClient())
+
+	for i := 0; i < 3; i++ {
+		if _, err := kv.Put("foo", "", lease.NoLease); err != nil {
+			t.Fatalf("couldn't put 'foo' (%v)", err)
+		}
+	}
+
+	if err := kv.Compact(2); err != nil {
+		t.Fatalf("couldn't compact kv space (%v)", err)
+	}
+	err := kv.Compact(2)
+	if err == nil || !strings.Contains(err.Error(), storage.ErrCompacted.Error()) {
```

Comment:
can you please add these types to v3rpc/errors.go and add the error handling to togRPCError func?